source/binder.hpp

```diff
@@ -15,6 +15,7 @@
 #define _INCLUDED_binder_hpp_
 
 
+#include "llvm/Config/llvm-config.h"
```

Comment:
Optional: again, is this include really needed or is this result of removed LLVM support?

Reply:
Ah, i see it now, - the lack indentation confused me. Please see two comments regarding it.